### DISCUSSION TITLE
Using reflex.testing library to test reflex apps.

### DISCUSSION THREAD

**Discussion by aden-collector** [reflex-dev/reflex#2941] [Q&A]
I am writting pytest to test the sdvp_web_app.py file.
[sdvp_web_app.txt](https://github.com/reflex-dev/reflex/files/14779880/sdvp_web_app.txt)
[test_stuff.txt](https://github.com/reflex-dev/reflex/files/14779882/test_stuff.txt)
This file is in the directory C:\sdvp_fastapi_web_server\sdv_web_app.  The pytest file is the attached test_stuff.txt file located in the C:\sdvp_fastapi_server\tests directory.  When I run the test, I get a %1 is not a valid win32 app in fixture function sdvp_web_app() in the AppHarness.create() function call.  If set the app_source parameter to sdvp_web_app, I get the "not a valid windows app" and the svdp_web_app.py file is overwritten with fixture source code.  If I set the app_name parameter to "sdvp_web_app", I get the same "invalid windows app" name error.  Finally, I tried running the upload test python file located in the repository and get the same window error.  Any ideas?

**Comment by masenf** [comment]
i'm able to repro this on windows 11. not sure what the problem is yet.

**Comment by masenf** [comment]
thanks for reporting. I pushed a fix in #2944
